spanner/test/spannerBackupTest.php

```diff
@@ -72,6 +72,9 @@ public static function setUpBeforeClass(): void
         if (!extension_loaded('grpc')) {
             self::markTestSkipped('Must enable grpc extension.');
         }
+        if (!getenv('GOOGLE_SPANNER_RUN_BACKUP_TESTS')) {
```

Comment:
nit: let's make this explicit (because `GOOGLE_SPANNER_RUN_BACKUP_TESTS=false` would still run the tests, for example). Also, our other env vars like this are explicit to either be "true" or "1":
```suggestion
        if ('true' !== getenv('GOOGLE_SPANNER_RUN_BACKUP_TESTS')) {
```